sql/session.proto

```diff
@@ -42,4 +42,10 @@ message Session {
   // Indicates that the above transaction is mutating keys in the
   // SystemDB span.
   optional bool mutates_system_db = 4 [(gogoproto.nullable) = false, (gogoproto.customname) = "MutatesSystemDB"];
+  oneof timezone {
+    // A time zone, local, or default (same as local).
```

Comment:
That calls for never allowing the use of Local/Default; I think that is the correct approach.